test/data/test_arco.py

```diff
@@ -57,7 +57,7 @@ def test_arco_fetch(time, variable):
     assert shape[3] == 1440
     assert np.array_equal(data.coords["variable"].values, np.array(variable))
     assert not np.isnan(data.values).any()
-    assert ARCO.available(time[0])
+    # assert ARCO.available(time[0])
```

Comment:
This is on purpose, the function is still tested below.

I added a not that when we fully jump to zarr 3.0, we need to update the available function to use async filesystem.